src/pages/index.js

```diff
@@ -121,13 +100,13 @@ const cardModalNew = new PopupWithForm(
   {
     popup: constants.newCardModal,
     handleFormSubmit: (newCardData) => {
-      api
-        .addNewCard(newCardData)
-        .then((newCardData) => {
-          addCard(newCardData);
-          cardModalNew.close();
-        })
-        .catch((err) => console.error(err));
+      function makeRequest() {
+        return api.addNewCard(newCardData).then((newCardData) => {
+          constants.cardGallery.prepend(createCard(newCardData));
```

Comment:
`constants.cardGallery.prepend`

Only class `Section` should add any card to the DOM.